java/change-notes/2021-06-25-apache-collections-maputils-keyvalue.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* Added models for the class MapUtils and the class keyvalue from Apache Commons Collections. This may lead to more results from any query using data-flow analysis where a relevant path uses one of these container types.
```

Comment:
"keyvalue" isn't a class, right? It's a package? Also, I believe we quote code like so:
```suggestion
* Added models for the class `MapUtils` and the class keyvalue from Apache Commons Collections. This may lead to more results from any query using data-flow analysis where a relevant path uses one of these container types.
```

Reply:
Or do you mean `KeyValue`?